src/did/Did.ts

```diff
@@ -47,13 +48,37 @@ export default class Did implements IDid {
   /**
    * @description Queries the [Did] from chain using the [identifier]
    *
-   * @param identifier the DIDs identifier
+   * @param identifier the DID identifier
    * @returns promise containing the [[Did]] or [null]
    */
   public static queryByIdentifier(identifier: string): Promise<IDid | null> {
     return queryByIdentifier(identifier)
   }
 
+  /**
+   * @description Gets the complete KILT DID from an [address] (in KILT, the method-specific ID is an address). Reverse of [[getAddressFromIdentifier]].
+   *
+   * @param address An address, e.g. "5CtPYoDuQQF...".
+   * @returns The associated KILT DID, e.g. "did:kilt:5CtPYoDuQQF...".
+   */
+  public static getIdentifierFromAddress(
+    address: IPublicIdentity['address']
+  ): IDid['identifier'] {
+    return getIdentifierFromAddress(address)
+  }
+
+  /**
+   * @description Gets the [address] from a complete KILT DID (in KILT, the method-specific ID is an address). Reverse of [[getIdentifierFromAddress]].
```

Comment:
Here I'm referring to an actual DID and not the SDK's Did class, so I didn't link DID to the SDK's documentation via [[]].